devito/types/sparse.py

```diff
@@ -76,9 +74,67 @@ def __shape_setup__(cls, **kwargs):
             shape = (glb_npoint[grid.distributor.myrank],)
         return shape
 
-    def _halo_exchange(self):
-        # no-op for SparseFunctions
-        return
+    def __fd_setup__(self):
+        """
+        Dynamically add derivative short-cuts.
+        """
+        return generate_fd_shortcuts(self.dimensions, self.space_order)
```

Comment:
👀

Reply:
It's just moved from above, space_order is zero this doesn't really do anything but needs to be implemented at some point